sdk/opa.go

```diff
@@ -145,8 +145,8 @@ func New(opts ...func(*OPA) error) (*OPA, error) {
 	return opa, nil
 }
 
-// StartNew convenience function to instantiate and start OPA in one command
-func StartNew(ctx context.Context, opts ...func(*OPA) error) (*OPA, error) {
+// StartOPAWaitUntilReady start new OPA and wait until plugins and bundles have been loaded
+func StartOPAWaitUntilReady(ctx context.Context, timeout time.Duration, opts ...func(*OPA) error) (*OPA, error) {
```

Comment:
I think commonly, we'd let the user control this from the context. So, we'd take in a `ctx` argument like before,
and add the select loop you've added to `AwaitReady`, but we would not expose the `timeout time.Duration` like this.